src/main/groovy/wooga/gradle/paket/unity/tasks/PaketUnityInstall.groovy

```diff
@@ -133,21 +142,17 @@ class PaketUnityInstall extends ConventionTask {
         fileTree.exclude("**/*.pdb")
         fileTree.exclude("**/Meta")
 
-        if (!getIncludeAssemblyDefinitions()) {
+        def excludeAsmDefs = !getIncludeAssemblyDefinitions()
+        if (excludeAsmDefs) {
```

Comment:
That just bloats the code. The check was readable fine enough before.

Reply:
Yes.